hooks/new

```diff
@@ -80,7 +82,8 @@ ask_for_database() {
       'Where would you like to house Cloud Foundry configuration and metadata?' \
       -o '[postgres-db]  External PostgreSQL Database' \
       -o '[mysql-db]     External MySQL Database' \
-      -o '[local-db]     an internal database node (not HA)'
+      -o '[local-db]     an internal database node (not HA)' \
+      -o '[local-ha-db]  internal HA database cluster'
```

Comment:
... and here

Reply:
Not sure why GitHub isn't labeling this as 'outdated'. The formatting has been corrected.

Reply:
looks good, despite what GH thinks.